src/index.ts

```diff
@@ -291,6 +298,23 @@ async function sendETH(ethWallet: ethers.Wallet, to: string, amount: BigNumber)
 
         console.log(`----------------------------------------------------------------------------`);
 
+        [transferAmount, l1feeAccountBalance] = await calculateTransferAmount(
+            l1feeAccountBalance,
+            blobOperatorBalance,
+            UPPER_BOUND_BLOB_OPERATOR_THRESHOLD,
+            LOWER_BOUND_BLOB_OPERATOR_THRESHOLD,
+            L1_ETH_TRANSFER_THRESHOLD
+        );
+        console.log(
+            `Amount which fee account can send to operator: ${ethers.utils.formatEther(transferAmount)} ETH;
```

Comment:
prob worth say that its blob operator?
will be hard to debug 2 same logs(see log on line 292)